packages/neuron-wallet/src/controllers/multisig.ts

```diff
@@ -214,7 +215,12 @@ export default class MultisigController {
     }
     const tx = result.json
     const lockHash = scriptToHash(addressToScript(fullPayload))
-    if (tx.transaction.inputs.every(v => v.lockHash !== lockHash)) {
+
+    if (tx.transaction && tx?.multisig_configs) {
```

Comment:
`optional` in `tx?.multisig_config` is redundant because `tx is non-null` is covered by `tx.transaction`